main.py

```diff
@@ -7,28 +7,44 @@
 
 # load model and processor
 model_id = "adept/fuyu-8b"
+
+tokenizer = AutoTokenizer.from_pretrained(model_id)
+
 processor = FuyuProcessor.from_pretrained(model_id)
 model = FuyuForCausalLM.from_pretrained(model_id, device_map="cpu", torch_dtype=torch.bfloat16)
 
 # prepare inputs for the model
-text_prompt1 = "<image>\nTesting\n"
+text_prompt1 = "<image>\nTesting\n<image>"
 url1 = "https://huggingface.co/adept/fuyu-8b/resolve/main/bus.png"
 image1 = Image.open(requests.get(url1, stream=True).raw)
 
-text_prompt2 = "What doesn this chart describe?\n<image>"
+text_prompt2 = "<image>What doesn this chart describe?\n<image>TestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTest<image>Other test"
```

Comment:
Nice test examples! Please also test a batch of 2 images/1 image/ no image to examine batch processing